### ISSUE TITLE
There is a large white space underneath repeated variable mappings in the harmonisation matrix on the variable detail page

### ISSUE THREAD

**Issue by BrendaHijmans** [molgenis/molgenis-emx2#4419]
Before you submit a new issue, please take a moment to check if it hasn't already been reported. This helps us streamline our efforts and ensures that your concern is addressed efficiently.

#### How to Check for Existing Issues
Search:
Use the GitHub search bar to look for keywords related to your issue. This includes error messages, feature names, or any relevant terms.

Issues Section:
Navigate to the "Issues" tab on our GitHub repository to see a list of reported problems and feature requests. Filter through open and closed issues to find similar cases.

**What version of EMX2 are you using (see footer)**
v11.25.0

**Describe the bug**
When not that many repeats of a harmonised variable are available (e.g. 3), there is a large white space beneath the harmonisation matrix on the variable detail page. Underneath, the details about mappings are visible.
![image](https://github.com/user-attachments/assets/5967b713-bb3d-4d35-a82b-144c14fe2f8d)

**To Reproduce**
1. Go to https://data-catalogue-acc.molgeniscloud.org/catalogue/ssr-catalogue/EHEN/variables/abd_circum_sdsWHO_t-ATHLETE-outcome_ath-ATHLETE?keys={%22name%22:%22abd_circum_sdsWHO_t%22,%22resource%22:{%22id%22:%22ATHLETE%22},%22dataset%22:{%22name%22:%22outcome_ath%22,%22resource%22:{%22id%22:%22ATHLETE%22}}}
2. See large white space below harmonisation matrix.

**Expected behavior**
The white space should be cut off at a certain margin.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by EleanorHyde-UMCG** [comment]
This bug is still an issue in prod. eg go to: https://data-catalogue.molgeniscloud.org/catalogue/catalogue/EHEN/variables/abd_circum_sdsWHO_t-ATHLETE-outcome_ath-ATHLETE?keys={%22name%22:%22abd_circum_sdsWHO_t%22,%22resource%22:{%22id%22:%22ATHLETE%22},%22dataset%22:{%22name%22:%22outcome_ath%22,%22resource%22:{%22id%22:%22ATHLETE%22}}} 

and see the large white space under the harmonisation matrix.

**Comment by chinook25** [comment]
According to the figma design this should indeed be removed.

**Comment by chinook25** [comment]
same as #3756 ?

**Comment by EleanorHyde-UMCG** [comment]
> same as [#3756](https://github.com/molgenis/molgenis-emx2/issues/3756) ?

no, not the same place but potentially the same problem. #3756 is the harmonisation matrix for multiple variables. This story ([renewed link here](https://molgeniscatalogue.org/EHEN/variables/abd_circum_sdsWHO_t-ATHLETE-outcome_ath-ATHLETE?keys={%22name%22:%22abd_circum_sdsWHO_t%22,%22resource%22:{%22id%22:%22ATHLETE%22},%22dataset%22:{%22name%22:%22outcome_ath%22,%22resource%22:{%22id%22:%22ATHLETE%22}}}) )   is the harmonisation matrix for a single variable with repeated values
